src/mixins/pluginHelper.js

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+
+export default {
+	methods : {
+
+		/**
+		 * Path to application file
+		 *
+		 * @param exp Extensionpoint
+		 * @param plugin Plugin
+		 */
+
+		getPlugin (ext) {
+			return _.find( this.$root.plugins, ['extend', ext]);
```

Comment:
I wonder if this should be `this.$plugins` ? that would imply that accessing plugins after initialization is desired. probably not useful, so maybe keep it as is

Reply:
I'll try to prototype them, but right now these are given to the root element upon instantiation so in order to access them, we need to address `this.$root`.